aws/resource_aws_ssm_maintenance_window.go

```diff
@@ -63,6 +64,12 @@ func resourceAwsSsmMaintenanceWindow() *schema.Resource {
 				Optional: true,
 			},
 
+			"schedule_offset": {
```

Comment:
Hi @sysamma 👋  Thank you for submitting this and its looking like a great start. Just a quick note that this new attribute will need documentation added in `website/docs/r/ssm_maintenance_window.html.markdown` as well. 👍

Reply:
Thanks for your kind words bflad7. I updated the docs file with the attribute